src/views/manage/ManageForm.jsx

```diff
@@ -96,14 +104,17 @@ const ManageForm = ({ id }) => {
 
   useEffect(() => {
     Object.keys(initalValues).map(name => register({ name: name }));
+    ['rapport_document', 'design_document'].map(name =>
```

Comment:
The `map` result isn't stored in a variable.

Reply:
not needed.